setup.py

```diff
@@ -50,8 +50,8 @@
             'numpy>=1.7',
             'pandas>=0.13.1',
             'datacache>=0.4.2',
-	    'pyfaidx',
-	    'memoized-property',
+            'pyfaidx>=0.3.4',
+            'memoized-property>=1.0.2',
```

Comment:
(Note that there isn't 100% parity at the moment, since `nose` and `pylint` don't seem necessary when a package is `pip` installed -- but I think it's fine to include them.)